bitnami/opensearch/README.md

```diff
@@ -94,20 +94,19 @@ initScriptsSecret=special-scripts-sensitive
 
 As it's described in the [official documentation](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/#register-repository), it's necessary to register a snapshot repository before you can perform snapshot and restore operations.
 
-This chart allows you to configure OpenSearch to use a shared file system to store snapshots. To do so, you need to mount a RWX volume on every OpenSearch node, and set the parameter `snapshotRepoPath` with the path where the volume is mounted. In the example below, you can find the values to set when using a NFS Persistent Volume:
+This chart allows you to configure snapshot repositories and snapshot policies in OpenSearch.
+A minimal configuration example looks like this:
 
 ```yaml
-extraVolumes:
-  - name: snapshot-repository
-    nfs:
-      server: nfs.example.com # Please change this to your NFS server
-      path: /share1
-extraVolumeMounts:
-  - name: snapshot-repository
-    mountPath: /snapshots
+snapshots:
+  enabled: true
+  persistence:
+    enabled: true
 snapshotRepoPath: "/snapshots"
 ```
 
+For details, please refer to the `snapshots.*` documentation below.
+
```

Comment:
Is this section of the README even relevant anymore? Should I remove it?

Reply:
@jotamartos Do you think I can remove it?

Reply:
If we have a configuration for the snapshots, you can remove the usage of extraVolumes and extraVolumeMounts

Reply:
I think I know what to do with the README as soon as we have landed in a decision regarding `snapshots.persistence.enabled` in the other thread. 👍

Reply:
Considering we have decided to keep `false,false` as the defaults, I think think it's helpful to the users if we document that both needs to be set to true when enabling snapshots. Thus, I think we stay with my current changes like they are now.